plugins/user-settings-backend/src/service/router.ts

```diff
@@ -40,6 +45,61 @@ export async function createRouter(options: {
     return credentials.principal.userEntityRef;
   };
 
+  // get multiple values
+  router.get('/multi', async (req, res) => {
```

Comment:
Maybe rather `/buckets/:bucket/multiget` or so.

URL limits eventually become a problem. The catalog batch fetch is a POST with `{ entityRefs: string[] }` for example, just to be safe.

Note also that the response should ideally be an object as well, e.g. `{ items: Array<JsonValue | null> }` or something like that, in case we want to add more stuff there such as etags or whatever.